bukuserver/templates/bukuserver/lib.html

```diff
@@ -31,17 +32,37 @@
   {% endif %}
 {% endmacro %}
 
-{% macro fetch_checkbox(checked=True) %}
+{% macro fetch_checkbox(checked=True, modal=False) %}
   <script>
     $('.admin-form [name=fetch]').remove();
-    $('.admin-form').append(
-      $(`<div class="form-group"><label style="display: block"><span class="col-md-2 text-right">{{ _gettext('Fetch') }} &nbsp; </span>`
-        +`<span class="col-md-10"><input type="checkbox" name="fetch"{% if checked %} checked{% endif %}></span></label></div>`));
+    $('.admin-form fieldset{% if modal %} .modal-footer{% endif %}').{% if modal %}prepend{% else %}append{% endif %}(
+      $(`<div class="form-group {{ 'mb-0' if modal else '' }}"{% if modal %} style="flex-grow: 1"{% endif %}>`
+        +`<label class="control-label {{ 'mb-0' if modal else '' }}">{{ _gettext('Fetch') }} &nbsp; </label>`
+        +`<input type="checkbox" name="fetch"{% if checked %} checked{% endif %}></div>`));
```

Comment:
Rearranging it similarly to how checkbox inputs are generated by `flask-admin` in Bootstrap4 templates